src/engine/base-engine.js

```diff
@@ -69,6 +93,10 @@ class BaseEngine {
    * @returns {Promise<void>} - The result promise
    */
   async initEngineServices(engineConfig) {
+    // const packageJson = JSON.parse(await fs.readFile('package.json'))
+    // this.version = packageJson.version
+    // TODO
+    this.version = '2.4.0'
```

Comment:
is this intended to be left and fixed in another PR?

Reply:
Fixed in another PR